hack/update-license-header.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+cd $(dirname $0)
+
+cd ../
+bin/license-header-checker -r -a -v -i vendor "hack/boilerplate.go.txt" . go
```

Comment:
maybe we need update `hack/boilerplate.go.txt` to remove the space prefix before License.

Reply:
it is another story, we can change it in another PR.